app/views/conferences/_call_for_tracks.haml

```diff
@@ -15,5 +15,5 @@
         left!
     %p.cta-button
       = link_to("Submit your request for track",
-        conference_program_tracks_path(conference_id),
+        new_conference_program_track_path(conference_id),
```

Comment:
JFYI; this conflicts with/supercedes #1865.

Reply:
that was unresponsive for very long so i thought may be could resolve that issue,
i will comment from the next time. :)